contents/handbook/strategy/overview.md

```diff
@@ -7,15 +7,29 @@ showTitle: true
 ## TL;DR
 Our mission is to increase the number of successful products in the world.
 
-We started by building an open-source product analytics platform with all the basic things you'd need to understand user behavior - funnels, trends, session recording, feature flags, and many more features. Within a year, we had thousands of customers using us and we started generating revenue.
+## Context
 
-We focused on our paid product earlier in 2021. We quickly hit a milestone for the first 5 reference customers and we've kept selling since. The next step on our journey is to figure out how to accelerate our top of funnel growth, while achieving larger order values from our Enterprise product.
+We started by building an open-source product operating system with many of the things you'd need to build a better product - product analytics, session recording, feature flags, experimentation, and a customer data platform to import/export and transform data. Within a year, we had thousands of customers using us and we started generating revenue.
 
-## Context
+We focused on our paid product in the Summer, 2021. We quickly hit a milestone for the first 5 reference customers. In 2022, we have figured out how to accelerate our top of funnel growth, and we realized we should focus on nailing Self Serve MidMarket ($20-70K ARR deals), whilst deepening the core product.
+
+Today, we're optimizing conversion to paid revenue, and carefully expanding the platform again, into adjacent use cases that will meet the needs for our _existing_ user profiles.
+
+We now add more companies each week than _any_ closed source SaaS rival, and we're the open source standard for all our applications.
+
+## General principles
+
+* For any company, nothing matters more than product market fit. If we get that right, it’ll be much easier for customer success, marketing or sales teams to succeed. As a result, we're 100% inbound. We invest in product, not cold-calling people that don't want to buy from us.
 
-We’ve grown a lot - tens of thousands of sign ups - but it’s clear that the many of our users use us *despite* a bad experience, with complex deployment and maintenance a particular issue. This is a great problem to have, as it means we’re solving a hair-on-fire problem - product analytics that you can self-host.
+* We are a _late mover_. All the products we build into the platform have product-market fit. This makes it faster to ship.
```

Comment:
do we have any principles around quality that we'd like to outline?

Reply:
Not to hand (this would be quite a fun page to create though).

I don't quite get how this concern relates to this specific point?

Reply:
Agreed it's a separate point